cmd/publishing-bot/publisher.go

```diff
@@ -50,8 +50,7 @@ type branchRule struct {
 
 // a collection of publishing rules for a single destination repo
 type repoRules struct {
-	dstRepo string
-	// publisher.go has assumption that src.repo is always kubernetes.
```

Comment:
@sttts not sure if I understand this assumption :-)

Reply:
There was some initial work to abstract the org, but never finished.